Automattic/jetpack
# VideoPress: Improve videopress invalid mime error

#### Changes proposed in this Pull Request:

* Re-implements the better error message for invalid Mime type implemented in https://github.com/Automattic/jetpack/pull/24757
* Add default error message

#### Other information:


#### Jetpack product discussion


NA

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Add a new block and upload a video with an invalid mime type (e.g .mkv)
* Check the specific error message

![Captura de tela de 2022-07-11 17-17-05](https://user-images.githubusercontent.com/971483/178351037-1971714c-a2c9-4352-bf3a-3943425f9183.png)




---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1202583706992749
